tests/integration-tests/constants.py

```diff
@@ -14,6 +14,11 @@
 
 OSU_BENCHMARK_VERSION = "5.7.1"
 
+ENCODE_CUSTOM_MUNGE_KEY = (
+    "lWXJDxgGhJxIVqLdbaycUICm12u0gHtcDFslGGxJlyLoVIQJFuskDfkK8wjvQfhT5pkeyuxA+vjgg9R+E+ftPV"
+    "TsVLHaf4bx3RmEfe30bZo79Yg+GhTRJRzV401/VaTlVEGFwMcJhmVKrXX/MbfnIdMwWNgCL8swUELbFOI4CG0K"
+)
+
```

Comment:
Let's think about a way to generate a base64 secret randomly on the fly, rather than having a constant secret.

Reply:
Done!